source/exe/main_common.cc

```diff
@@ -8,7 +8,10 @@
 #include "common/stats/stats_impl.h"
 #include "common/stats/thread_local_store.h"
 
+#ifdef ENVOY_HOT_RESTART
 #include "exe/hot_restart.h"
+#endif
```

Comment:
I left it in now that it's under common. I could add an `#ifndef ENVOY_HOT_RESTART`, but I don't think it hurts anything to leave it.